src/ml_board/frontend/Dashboard/src/worker_socket/DataTypes.ts

```diff
@@ -20,7 +20,14 @@ export interface BufferedDataFromSocket extends JSON {
 }
 
 export interface DataToRedux {
+    tableHeaders?: string[],
     tableData: Row[],
     chartsUpdates: ChartUpdate[],
     status?: any,
+}
+
+export interface UpdatesObject {
```

Comment:
can be renamed to something better... like DashboardTableObjects or maybe something else

Reply:
but it doesn't just hold the updates of the Dashboard, it actually the graph updates too ?! 
https://github.com/mlgym/mlgym/blob/67b58a7ccf8b0d5423231f4094b5891799943e53/src/ml_board/frontend/Dashboard/src/worker_socket/DataTypes.ts#L29-L33

this is later processed into a `DataToRedux` object
https://github.com/mlgym/mlgym/blob/67b58a7ccf8b0d5423231f4094b5891799943e53/src/ml_board/frontend/Dashboard/src/worker_socket/DataTypes.ts#L22-L27

using this
https://github.com/mlgym/mlgym/blob/67b58a7ccf8b0d5423231f4094b5891799943e53/src/ml_board/frontend/Dashboard/src/worker_socket/MainThreadCallbacks.ts#L65-L74

Reply:
Okay got it!